OpenIPSL/Electrical/Renewables/PSSE/PlantController/BaseClasses/BaseREPC.mo

```diff
@@ -0,0 +1,161 @@
+within OpenIPSL.Electrical.Renewables.PSSE.PlantController.BaseClasses;
+partial model BaseREPC
+  "Base class for the plant controller of the WECC generic renewable models"
+
+  import Complex;
+  import Modelica.ComplexMath.arg;
+  import Modelica.ComplexMath.real;
+  import Modelica.ComplexMath.imag;
+  import Modelica.ComplexMath.conj;
+  import Modelica.Blocks.Interfaces.*;
+  parameter OpenIPSL.Types.ApparentPower M_b=SysData.S_b "Machine base power" annotation (Dialog(group="Power flow data"));
+  extends OpenIPSL.Electrical.Essentials.pfComponent(
+    final enabledisplayPF=false,
+    final enablefn=false,
+    final enableV_b=false,
+    final enableangle_0=true,
+    final enablev_0=true,
+    final enableQ_0=true,
+    final enableP_0=true,
+    final enableS_b=true);
+
+  parameter Boolean vcflag=true "Reactive dropp (False) or line drop compensation (True)" annotation (Dialog(tab="Control"));
+  parameter Boolean refflag=true "Plant-level reactive power (False) or voltage control (True)" annotation (Dialog(tab="Control"));
+  parameter Boolean fflag=true "Governor response disable (False) or enable (True)" annotation (Dialog(tab="Control"));
+
+  Modelica.Blocks.Interfaces.RealInput Qref annotation (Placement(
+        transformation(extent={{-240,90},{-200,130}}),  iconTransformation(
+          extent={{-240,90},{-200,130}})));
+  Modelica.Blocks.Interfaces.RealInput Plant_pref annotation (Placement(
+        transformation(extent={{-240,20},{-200,60}}), iconTransformation(
+          extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-220,40})));
+  Modelica.Blocks.Interfaces.RealInput Freq annotation (Placement(
+        transformation(extent={{-240,-60},{-200,-20}}), iconTransformation(
+          extent={{-240,-60},{-200,-20}})));
+  Modelica.Blocks.Interfaces.RealInput Freq_ref
+    "Connector of Real input signal 2" annotation (Placement(
+        transformation(extent={{-240,-130},{-200,-90}}),
+        iconTransformation(extent={{-240,-130},{-200,-90}})));
+  Modelica.Blocks.Interfaces.RealOutput Qext(start=q0) "Connector of Real output signal"
```

Comment:
That descriptions string is useless since it does not tell you what hat connector is representing.

Reply:
I've fixed this no?

Reply:
Well kind of. Let me comment on the latest.